src/procedures/procedures.11tydata.yaml

```
@@ -286,6 +286,186 @@ procedureMigrateZigbeeZHAOnly:
       image: z2m-backup-restore-05.png
       content: |
         <ul>
-        <li class="info">All migrated devices have a new entity ID. This means you will need to manually reconfigure automations and dashboard cards.</li>
+        <li class="info">All migrated devices have new entity IDs. This means you will need to manually reconfigure automations and dashboard cards.</li>
+          <li class="info">The names that you manually assigned to your devices won't have been migrated. You will have to rename the devices again.</li>
+        </ul>
+procedureMigrateConBee:
+  title: Migrate a deCONZ network to SkyConnect using ZHA
+  steps:
+    - title: View the firmware version of the stick
+      image: conbee-update-05.png
+      content: |
+        <ol>
+          <li>In the deCONZ app, under <b>Settings</b>><b>Gateway</b>, check the firmware version. It should be version 26720700.
+            <ul>
+              <li>If the version is correct, skip the next two steps and continue with the step on stopping the deCONZ add-on.</li>              
+            </ul>
+          </li>
+          <li>If the version is not up-to-date, select <b>Update to 26720700</b>.</li>
+        </ol>
+    - title: Follow the wizard
+      image: conbee-update-06.png
+      content: |
+        <ul>
+          <li>Select <b>Create backup</b> and let the wizard guide you through the update process.</li>
+        </ul>
+    - title: Update complete
+      image: conbee-update-14.png
+      content: |
+        <ul>
+          <li>Once the update is complete, you will see the new firmware version.</li>
+        </ul>
+    - title: Stop the deCONZ add-on
+      image: conbee-migrate-zha-02.png
+      content: |
+        <ul>
+          <li>Under <b>Settings</b>><b>Add-ons</b>, in the <b>Zigbee2MQTT</b> add-on, select <b>Stop</b>.</li>
```

```suggestion
          <li>Under <b>Settings</b>><b>Add-ons</b>, in the <b>deCONZ</b> add-on, select <b>Stop</b>.</li>
```

Also, this can be a My-Link directly to the Add-on. The add-on name is what the URL shows when in the add-on page (in this case `core_deconz`, so the link is https://my.home-assistant.io/redirect/supervisor_addon/?addon=core_deconz).

---

Also, please ask the user to disable on startup, otherwise they will run into issue after the next reboot :cold_sweat: